src/components/fin/metricSlider/heading/HeadingButtons.tsx

```diff
@@ -45,9 +43,15 @@ export default function HeadingButtons({ size = 'md', endPage }: Props) {
 
   const timeoutRef = useRef<NodeJS.Timeout | null>(null)
 
+  // Detect with userAgent
+  const isMobile =
```

Comment:
On est sur de tout catch avec ça ? La lib is-mobile est un peu plus étoffée https://github.com/juliangruber/is-mobile/blob/main/index.js

Reply:
Okay ça me va ! J'ajoute ça